flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/TestBaseUtils.java

```diff
@@ -408,10 +408,20 @@ public static void compareKeyValuePairsWithDelta(String expectedLines, String re
 		}
 	}
 
+	public static <X extends Comparable<? super X>> void compareResultCollections(List<X> expected, List<X> actual) {
```

Comment:
Please try to avoid modifying classes outside of `flink-table` unless really necessary.